bgpd/bgp_mplsvpn.c

```diff
@@ -1730,10 +1755,12 @@ void vrf_unimport_from_vrf(struct bgp *to_bgp, struct bgp *from_bgp,
 	if (to_bgp->vpn_policy[afi].import_vrf->count == 0) {
 		UNSET_FLAG(to_bgp->af_flags[afi][safi],
 			   BGP_CONFIG_VRF_TO_VRF_IMPORT);
-		ecommunity_free(&to_bgp->vpn_policy[afi].rtlist[idir]);
+		if (to_bgp->vpn_policy[afi].rtlist[idir])
```

Comment:
To avoid accessing already freed ecommunity. When a source vrf instance is removed, we free the ecommunity. Later if user decided to remove 'no import vrf' from target vrf instance, avoid freeing ecommunity instead simply unset the configuration flag.